spec/base_spec_helper.rb

```diff
@@ -131,6 +131,17 @@
     end
   end
 
+  # Appends Stripe gem version to VCR cassette directory with ':stripe_version' flag
+  config.around(:each, :stripe_version) do |example|
+    stripe_version = "Stripe-v#{Stripe::VERSION}"
+    VCR.configure do |c|
+      c.cassette_library_dir = "spec/fixtures/vcr_cassettes/#{stripe_version}"
+      # record: needs to be set to :once to record a new reply (cassette)
+      c.default_cassette_options = { record: :none }
```

Comment:
Variable name `c` is okay but `vcr_config` would be better, I think.
```suggestion
    VCR.configure do |vcr_config|
      vcr_config.cassette_library_dir = "spec/fixtures/vcr_cassettes/#{stripe_version}"
      # record: needs to be set to :once to record a new reply (cassette)
      vcr_config.default_cassette_options = { record: :none }
```

And I would only set it to `:none` when we are running this on CI.
```suggestion
    VCR.configure do |vcr_config|
      vcr_config.cassette_library_dir = "spec/fixtures/vcr_cassettes/#{stripe_version}"
      # record: needs to be set to :once to record a new reply (cassette)
      vcr_config.default_cassette_options = { record: :none } if ENV["CI"]
```